com.unity.render-pipelines.universal/Editor/ScriptableRendererDataEditor.cs

```diff
@@ -104,6 +104,18 @@ private bool GetCustumTitle(Type type, out string title)
             return false;
         }
 
+        private bool GetTooltip(Type type, out string tooltip)
+        {
+            var attribute = type.GetCustomAttribute<TooltipAttribute>();
+            if (attribute != null)
+            {
+                tooltip = attribute.tooltip;
+                return attribute != null;
+            }
+            tooltip = "";
```

Comment:
I would recommend to use string.Empty
```suggestion
            tooltip = string.Empty;
```

Reply:
fixed